src/python/pants/backend/project_info/filedeps_test.py

```diff
@@ -31,36 +24,19 @@ def rules(cls):
 
     @classmethod
     def target_types(cls):
-        return [GenericTarget, Resources, PythonLibrary]
+        return [MockTarget, ProtobufLibrary]
 
-    @classmethod
-    def alias_groups(cls) -> BuildFileAliases:
-        return BuildFileAliases(
-            targets={
-                "target": TargetV1,
-                "resources": ResourcesV1,
-                "java_library": JavaLibraryV1,
-                "java_thrift_library": JavaThriftLibraryV1,
-                "jvm_app": JvmAppV1,
-                "jvm_binary": JvmBinaryV1,
-                "scala_library": ScalaLibraryV1,
-                "python_library": PythonLibraryV1,
-            },
-        )
-
-    def create_python_library(
+    def setup_target(
         self,
         path: str,
         *,
         sources: Optional[List[str]] = None,
-        dependencies: Optional[List[str]] = None
+        dependencies: Optional[List[str]] = None,
     ) -> None:
-        self.create_library(
```

Comment:
`self.create_library` doesn't work well in a V2 world, as it creates a V1 target. 

Instead, `TestBase.add_to_build_file` has been working quite nicely. It's a little less generalized, and I think that's a good thing here.